README.rst

```diff
@@ -102,3 +102,34 @@ and adds a link to it in the header of any other view:
   <link rel="amphtml" href="${context/absolute_url}/@@amp">
 
 The body text is processed to remove invalid elements or to replace them by the corresponding AMP components.
+
+Tracking pixels
+^^^^^^^^^^^^^^^
+
+AMP implements tracking pixels using the ``<amp-pixel>`` tag;
+to add support for this feature you need to write an adapter that implements the ``IAMPPixelProvider`` interface like this:
+
+.. code-block:: xml
+
+    <adapter factory="my.package.adapter.AMPPixelProvider" />
+
+
+.. code-block:: python
+
+    from collective.behavior.amp.behaviors import IAMP
+    from collective.behavior.amp.interfaces import IAMPPixelProvider
+    from zope.component import adapter
+    from zope.interface import implementer
+
+    @implementer(IAMPPixelProvider)
+    @adapter(IAMP)
+    class AMPPixelProvider(object):
```

Comment:
you're basically proposing the same thing @rodfersou told me yesterday, but IMO calling a view will have worst performance than using an adapter and we already have performance issues on this `@@amp` view.

on the other side, it is possible indeed to have different tracking pixels for different content types and it's easier: I just have to check inside the adapter which content type is the object and act consequently:

```python
def pixel(self):
    if self.context.portal_type == 'foo':
        return u'<amp-pixel src="https://foo.com/tracker" layout="nodisplay"></amp-pixel>'
    elif self.context.portal_type == 'bar':
        return u'<amp-pixel src="https://bar.com/tracker?RANDOM" layout="nodisplay"></amp-pixel>'
```
our current use case, for instance, involves different tracking pixels for different content authors: I can check in the adapter who is the author of the piece of content and then return different strings for each one.

that's why I choose the adapter.